README.md

```diff
@@ -79,7 +79,7 @@ scalatest_toolchain()
 
 This will load the `rules_scala` repository at the commit sha
 `rules_scala_version` into your Bazel project and register a [Scala
-toolchain](#scala_toolchain) at the default Scala version (2.12.14)
+toolchain](docs/scala_toolchain.md) at the default Scala version (2.12.14)
```

Comment:
Not really important and can be ignored but still: maybe change link text to `scala_toolchain` so it will be consistent with other usages